prime-router/metadata/hl7_mapping/ORU_R01/base/patient-visit.yml

```diff
@@ -0,0 +1,92 @@
+name: ORU-R01-patient-visit
+constants:
+  hl7PV1Field: /PATIENT_RESULT/PATIENT/VISIT/PV1
+elements:
+  - name: set-id
+    value: [ '''1''' ]
+    hl7Spec: [ '%{hl7PV1Field}-1' ]
+
+  - name: patient-class
+    condition: '%resource.class.code.exists()'
+    value: [ '%resource.class.code.getV3ActCode()' ]
```

Comment:
I know the Linux4Health library calls it the V3 Act code but tbh I don't know that its the clearest description of what this represents. can we call it `getPatientClass` or something similar since thats what the element name is?

Reply:
Yes, change to something like getPatientClass.

Reply:
Done